sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkGetColumnsOperation.scala

```diff
@@ -58,9 +59,12 @@ private[hive] class SparkGetColumnsOperation(
 
   override def runInternal(): Unit = {
     // Do not change cmdStr. It's used for Hive auditing and authorization.
-    val cmdStr = s"catalog : $catalogName, schemaPattern : $schemaName, tablePattern : $tableName"
-    val logMsg = s"Listing columns '$cmdStr, columnName : $columnName'"
-    logInfo(s"$logMsg with $statementId")
+    val cmdStr = log"catalog : ${MDC(CATALOG_NAME, catalogName)}, " +
+      log"schemaPattern : ${MDC(SCHEMA_NAME, schemaName)}, " +
+      log"tablePattern : ${MDC(TABLE_NAME, tableName)}"
+    val logMsg = log"Listing columns '${MDC(COMMAND, cmdStr)}, " +
+      log"columnName : ${MDC(COLUMN_NAME, columnName)}'"
+    logInfo(log"${MDC(LOG_MESSAGE, logMsg)} with ${MDC(STATEMENT_ID, statementId)}")
```

Comment:
This is wrong...let's flatten eveything into one log entry instead of nested logs.

Reply:
Perhaps we can improve the logic of MDC to throw exceptions in this case. Let me give it a try.

Reply:
Thanks for the comment. Just updated.